eul/StatusBar/StatusBarItem.swift

```diff
@@ -102,6 +112,11 @@ class StatusBarItem: NSObject, NSMenuDelegate {
         super.init()
 
         statusBarMenu.delegate = self
+        if preferenceStore.appearanceMode == .light {
+            statusBarMenu.appearance = NSAppearance(named: .aqua)
+        } else {
+            statusBarMenu.appearance = NSAppearance(named: .darkAqua)
+        }
```

Comment:
maybe re-use `changeNSWindowColorScheme`?

Reply:
You mean like a ternary operator for the parameter like 
`NSAppearance(named: preferenceStore.appearanceMode == .light ? .aqua : .darkAqua)`. Is this okay?

Reply:
I mean the function itself - like you can call `changeNSWindowColorScheme()` here

Reply:
Well the `changeNSWindowColorScheme` takes in a `NSAppearance`, so I added another function to read from preferences, and call `changeNSWindowColorScheme` with correct parameters